### DISCUSSION TITLE
Recettage appli

### DISCUSSION THREAD

**Discussion by SteffMangin** [JMPivette/evavelo#19] [Q&A] [answered]
Bonjour @JMPivette ,
Avec un peu de retard, je viens de faire des opérations de contrôle sur l'appli baptisée cette semaine par Camille 'ÉVA-Scan' ;-).
J'ai quelques remarques/interrogations. Je te les livre en pièce jointe (dans le doc) mais si c'est plus simple, on peut échanger par téléphone.
[2021_01_22_Recettage_EVA-Scan.docx](https://github.com/JMPivette/evavelo/files/5854625/2021_01_22_Recettage_EVA-Scan.docx)
[2021-01-22_Fichier_recettage4.xlsx](https://github.com/JMPivette/evavelo/files/5854626/2021-01-22_Fichier_recettage4.xlsx)
[2021-01-22_Fichier_recettage5.xlsx](https://github.com/JMPivette/evavelo/files/5854630/2021-01-22_Fichier_recettage5.xlsx)
[2021-01-22_Fichier_recettage6.xlsx](https://github.com/JMPivette/evavelo/files/5854631/2021-01-22_Fichier_recettage6.xlsx)

J'attends tes retours avant de poursuivre.

J'ai bien en tête de te fournir un fichier avec les données de comptages. Mais ce que j'ai de l'enquête sur La Vélodyssée et particulièrement cracra. J'ai voulu te faire un export et te préparer un fichier type mais j'ai un souci avec Eco-Visio. Je suis en attente d'un retour de Laure.

A ta dispo pour échanger si besoin. 

A+

**Comment by JMPivette** [comment]
Merci @VeloTerritoires pour ces retours.

Je vais étudier tout ça et décomposer en différents bugs.

A première vue, sans être rentré dans le détails, voici quelques éléments de réponses sur des problèmes potentiels:
* Pour l'instant je fais mon analyse en me basant sur l'onglet `enquetes_post_traitement` et non `
enquetes_saisies`. Je partais du principe qu'il y avait les mêmes informations (sans compter les colonnes additionnelles). Je pourrais rajouter une vérification entre ces 2 onglets. (tout comme entre `comptages_manuels`et `comptage_man_post_traitements`) Tu en penses quoi?

* Je viens de vérifier l'affectation de `categorie_corrigee` et en effet pour l'instant je ne mettais à jour que les cas pour lesquels il y avait une différence entre `categorie_visuelle`et `categorie questionnaire`. Les autres valeurs restaient celle du fichier d'input sans modification.
Je vais corriger cela.

* L'onglet `comptage`de l'output est utilisé pour la mise à jour de `categorie_visuelle_cycliste_corrige`de l'onglet `comptages_man_post_traitements`. Peut-être que j'ai mal interpreté cette partie dans la méthodologie. Je vais regarder plus en détail.

**Reply by SteffMangin** [reply]
Bonjour @JMPivette ,

Sur ton commentaire précédent, je suis désolée, j'ai peur de mal interpréter le point 1. Est-ce que l'on peut en parler par tél ?

Pour le point 2, il faut considérer que dans l'input la colonne [categorie_corrige] est vide. Attention, je ne sais pas si  c'est pareil dans ton code mais dans ton commentaire, tu as mentionné le champ [categorie_corrig**ee**] alors que c'est [categorie_corrige] .

Pour le point 3, je ne comprends pas car il n'y a pas d'onglet 'comptage'. Est-ce que tu parles de l'onglet comptages_manuels ou de l'onglet comptages_automatiques ?

A+

**Comment by JMPivette** [comment, marked as answer]
Retours dispatchés en différentes "issues":
* #21 
* #23 
* #22 
* #25 
* #20
